### PR TITLE
View users add entity/jira wp 374

### PR DESCRIPTION
**Pull request by davidmtacc**
## Overview
Added entity to initial table and to view user modal as requested
…

## Related

…

## Changes
Updated get_users db query, modified table and modal by replacing org_name with entity_name 
…

## Testing

1. New table shows Org and entity under Organizaton column

## UI

…
Screenshot 2023-11-07 at 12 37 20 PM

### CODE REVIEW

File: apcd-cms/src/apps/view_users/templates/view_user_modal.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h4 class="modal-title">Details for User: {{r.user_name}} ({{r.user_id}})</h4>
                 <dt class="c-data-list__key">Email</dt>
                 <dd class="c-data-list__value">{{r.user_email}}</dd>
                 <dt class="c-data-list__key">Organization</dt>
```
Comment (sophia-massie) [inline apcd-cms/src/apps/view_users/templates/view_user_modal.html:28]:
```suggestion
                <dt class="c-data-list__key">Entity Organization</dt>
```
To match the other admin tables and to be clear which field this is pulling

File: apcd-cms/src/apps/view_users/views.py [hunk 2]
```diff
@@ -64,14 +64,14 @@ def _set_user(usr):
...
                     'created_at': usr[5],
                     'updated_at': usr[6],
                     'notes': usr[7],
                     'status': 'Active' if usr[8] else 'Inactive',
                     'user_number': usr[9],
                     'role_name': usr[10],
-                    'org_name_no_parens': usr[4].replace("(", "").replace(")", ""),  # just for filtering purposes
+                    'entity_name_no_parens': usr[4].replace("(", "").replace(")", "") if usr[4] else "Not Applicable",  # just for filtering purposes
                     'active': usr[8],
                 }
```
Comment (sophia-massie) [inline apcd-cms/src/apps/view_users/views.py:77]: For line 78: 
`context['header'] = ['User ID', 'Name', 'Entity Organization', 'Role', 'Status', 'User Number', 'See More']`

To match other admin tables and be clear which field this is pulling

**Review by sophia-massie — CHANGES_REQUESTED** [review]
Looks great! One thing I noticed that was missing is that the edit user modal needs to also change to pull the r.entity_name and have the title of that field be `<dt class="c-data-list__key">Entity Organization</dt>` to line up with other admin table and modal labels. It's blank right now.

Screenshot 2023-11-07 at 5 13 38 PM

File: apcd-cms/src/apps/view_users/templates/view_user_modal.html (hunk 1 above)
Comment (davidmtacc) [reply apcd-cms/src/apps/view_users/templates/view_user_modal.html]: Changed name field title to Entity Organization and added the field to the edit modal.

File: apcd-cms/src/apps/view_users/views.py (hunk 2 above)
Comment (davidmtacc) [reply apcd-cms/src/apps/view_users/views.py:77]: Changed to Entity Organization

**Review by edmondsgarrett — APPROVED** [review]
Looks good!

**Review by sophia-massie — APPROVED** [review]

**Pull request merged**